docs/web-chat/identity-users.mdx

```diff
@@ -0,0 +1,697 @@
+---
+sidebar_position: 45
+title: User Hub
```

Comment:
We are using `Identity User Management` in SDK X dev docs. We should keep it consistent in both SDK X & web chat dev docs.
https://developers.helpshift.com/sdkx_ios/identity-users/